bindings/C/adios2/c/adios2_c_adios.h

```diff
@@ -46,7 +46,7 @@ adios2_adios *adios2_init_config_mpi(const char *config_file, MPI_Comm comm);
 
 #else
 #define adios2_init() adios2_init_serial()
-#define adios2_init_config(config_file) adios2_init_config_seria(config_file)
```

Comment:
Ouch.  We may need to push this into the release branch too.

Reply:
Seems like no one uses the non-MPI version 🤣